de.dlr.sc.virsat.apps.feature/pom.xml

```diff
@@ -18,7 +18,7 @@ SPDX-License-Identifier: EPL-2.0
     <relativePath>../maven/pom.xml</relativePath>
     <groupId>de.dlr.sc.virsat</groupId>
     <artifactId>de.dlr.sc.virsat.parent</artifactId>
-    <version>4.17.1-SNAPSHOT</version>
+    <version>4.17.0-SNAPSHOT</version>

```

Comment:
Major concern. This is reverting back all the changes from development. E.g. switching back from 4.17.1 to 4.17.0